tap_copper/schemas/custom_fields.json

```diff
@@ -0,0 +1,70 @@
+{
+  "type": [
+    "null",
+    "object"
+  ],
+  "properties": {
+    "id": {
+      "type": [
+        "null",
```

Comment:
you can remove this `null` - the primary key field should basically never be null! If it is, Stitch will be unable to load the record. Better for us to catch this in the tap IMO